docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -1273,6 +1290,56 @@ type of a variable::
     {{ myvar | type_debug }}
 
 
+Computer Therory Assertions
+```````````````````````````
+
+There are functions ``human_readable`` and ``human_to_bytes``.
```

Comment:
Could you add a use case here? I'm not sure I understand these features completely, so this is a guess, but something like:

"The functions `human_readable` and `human_to_bytes` let you test your playbooks to make sure you are using the right size format in your tasks - that you're providing Byte format to computers and human-readable format to people."

Reply:
Sure I will add them.